apps/next/src/components/Forms/FormField.tsx

```diff
@@ -98,12 +102,14 @@ export const FormField = ({
           >
             <Stack component="label" onClick={() => setIsEditing(true)}>
               <InvisibleFieldInput
+                type={type}
```

Comment:
Can we make it such that the error message is only shown if the field was touched/focused?
This way we won't be displaying this scary all-red form as soon as the user enters the page.

Quick idea:

```tsx
  const [wasTouched, setWasTouched] = useState(false);

  const showPrompt = Boolean(value || isEditing || (wasTouched && required));

  const handleChange = useCallback(
    (e: React.ChangeEvent<HTMLInputElement>) => {
      setWasTouched(true);
      onChange(e.target.value);
    },
    [onChange],
  );

  // then use `handleChange` on the input field
  // and only render `error` / change color to red if `wasTouched` is true
```